src/components/Trade/hooks/useSwapper/useSwapperV2.tsx

```diff
@@ -113,12 +113,10 @@ export const useSwapper = () => {
 
   const getReceiveAddressFromBuyAsset = useCallback(
     async (buyAsset: Asset) => {
-      return getReceiveAddress({
-        asset: buyAsset,
-        wallet,
-        bip44Params: buyAccountMetadata.bip44Params,
-        accountType: buyAccountMetadata.accountType,
-      })
+      if (!buyAccountMetadata) return
+      const { accountType, bip44Params } = buyAccountMetadata
+      if (!bip44Params) return
```

Comment:
is there a reason/use case for no `bip44Params` on the account metadata? Seems like it should be a required field to me